server/jetstream_api.go

```diff
@@ -2351,7 +2352,7 @@ func (s *Server) jsMsgGetRequest(sub *subscription, c *client, subject, reply st
 
 	subj, hdr, msg, ts, err := mset.store.LoadMsg(req.Seq)
 	if err != nil {
-		resp.Error = jsError(err)
+		resp.Error = jsNoMessageFoundErr
```

Comment:
not sure, i mean LoadMsg can fail for many reasons, perhaps LoadMsg should return a specific error for not found vs other kinds of failure - if it can detect it, then we can have 404 and 500 appropriately?

Reply:
Most of the others are captured above. This is what we do for a pull consumer also, essentially turn everything into a `not found` error.

Reply:
Ok. Consistency matters lost then. LGTM